website/docs/components/rich-tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,285 @@
+```
+
+Similarly, a typographic class (or a custom CSS class) can be applied to a parent container.
+
+#### Whitespace issues
+
+There may be cases in which the tooltip text is preceded/followed by non-whitespace characters. In this case extra whitespace could appear between these characters and the toggle text. Unfortunately he only solution we have found is to remove every whitespace before/after the `<Hds::RichTooltip>` tag, and before/after the `<RT.Toggle>` and `<RT.Bubble>` tags:
+
+```handlebars
+<Hds::Text::Body @tag="p" @size="300">
+  Lorem "<Hds::RichTooltip as |RT|><RT.Toggle @isInline={{true}} @text="ipsum dolor" /><RT.Bubble>
+      <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+      <Hds::Text::Body @tag="p" @size="200">Some descriptive information</Hds::Text::Body>
+    </RT.Bubble></Hds::RichTooltip>" sit amet consectetur adipiscing elit.
+</Hds::Text::Body>
+```
+
+### Interactivity
+
+By default the visibility of the tooltip is toggled via "soft" event listeners (hover/focus) applied to the "toggle" container. _Notice: from a purely technical standpoint, the events are `mouseEnter/Leave` and `focusIn/Out`._
+
+To change this behavior and opt for a more explicit user interaction enabling "click" events (in this case the "soft" events are disabled) set the argument `@enableClickEvents` to `true`:
+
+```handlebars
+<Hds::RichTooltip @enableClickEvents={{true}} as |RT|>
+  <RT.Toggle @size="medium" @text="More info" @icon="info" />
+  <RT.Bubble>
+    <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" @size="200">Some descriptive information</Hds::Text::Body>
+  </RT.Bubble>
+</Hds::RichTooltip>
+```
+
+Independent of which interaction is used, the tooltip can be dismissed by clicking outside of the component or with the `esc` key (this "light dismiss" behavior is automatically provided by the [Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API/Using#auto_state_and_light_dismiss)).
+
+### Placement
+
+By default the tooltip is shown below the toggle, visually centered. It's possible to change the initial position of the tooltip using the `@placement` argument:
+
+```handlebars
+<Hds::RichTooltip as |RT|>
+  <RT.Toggle @size="medium" @text="Lorem ipsum" @icon="info" />
+  <RT.Bubble @placement="top-start">
+    <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" @size="200">Some descriptive information that spans multiple lines</Hds::Text::Body>
+  </RT.Bubble>
+</Hds::RichTooltip>
+```
+
+### Collision detection
+
+The RichTooltip component automatically adapts its alignment depending on its position relative to the viewport to avoid "collisions" with the browser window boundaries. This means that when an end-user scrolls the page, or resizes the browser, the position of the tooltip on the page dynamically adapts to these changes (along the two axes, main and secondary).
```

Comment:
```suggestion
The RichTooltip component automatically adapts its alignment depending on its position relative to the viewport to avoid collisions with the browser window boundaries. This means that when an end-user scrolls the page, or resizes the browser, the position of the tooltip on the page dynamically adapts to these changes (along the two axes, main and secondary).
```